Face/VerifyFaceToFace/Program.cs

```diff
@@ -8,7 +8,7 @@ namespace VerifyFaceToFace
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async void Main(string[] args)
```

Comment:
Same, this will not pass build.
It should be like `FindSimilar` project:
`public static async Task Detect()`
And call `Detect.Wait();` in
`public static void Main(string[] args)`

Reply:
done